droidlet/dashboard/web/src/components/Interact/Question.js

```diff
@@ -560,6 +566,7 @@ class Question extends Component {
     var data = {
       action_dict: this.state.action_dict,
       parsing_error: this.state.parsing_error,
+      perception_error: this.state.perception_error,
```

Comment:
There's a bug on line 495 of this file, `vision_error: true` should be `perception_error: true`, which breaks vision error logging.